source/Camera/Advanced/CaptureHDRLoop/CaptureHDRLoop.cpp

```diff
@@ -19,10 +35,12 @@ int main()
         auto camera = zivid.connectCamera();
 
         const size_t captures = 3;
+        const auto cameraModel = getCameraModel(camera);
+
```

Comment:
Nitpick - change the order of this and the previous line of code. Get rid of the empty line after this line.

Reply:
Done